toolsrc/src/vcpkg/export.chocolatey.cpp

```diff
@@ -181,7 +181,7 @@ if (Test-Path $installedDir)
             Strings::inplace_replace_all(norm_version, '-', '.');
             Strings::inplace_replace_all(norm_version, '_', '.');
             norm_version += chocolatey_options.maybe_version_suffix.value_or("");
-            packages_version.insert(std::make_pair(binary_paragraph.spec.name(), norm_version));
+            packages_version.insert(std::make_pair(binary_paragraph.spec, norm_version));
```

Comment:
Map insert should never make_pair as that introduces an unnecessary copy into the temporary pair

```suggestion
            packages_version.emplace(binary_paragraph.spec, norm_version);
```